app/jobs/send_closing_notification_job.rb

```diff
@@ -0,0 +1,7 @@
+class SendClosingNotificationJob < ApplicationJob
+  def perform(users, content, procedure)
+    users.each do |user|
+      Expired::MailRateLimiter.new().send_with_delay(UserMailer.notify_after_closing(user, content, @procedure))
+    end
```

Comment:
Je trouve ca un poil casse gueule de charger tous les users, peut etre qu'on peut les loader petit a petit ?

```suggestion
  def perform(user_ids, content, procedure)
    User.where(id: user_ids).find_each do |user|
      Expired::MailRateLimiter.new().send_with_delay(UserMailer.notify_after_closing(user, content, @procedure))
    end
```